### PR TITLE
Powershell and Python code to get NetBackup alerts.

### PR DESCRIPTION
**Pull request by ashutosh1211**
Powershell and Python code snippets to get NetBackup alerts.

### CODE REVIEW

File: snippets/powershell/Get-NB-Alerts.ps1 [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+.SYNOPSIS
+This sample script demonstrates the use of NetBackup REST API for listing the alerts.
+
+.DESCRIPTION
+This script will get a list of netbackup alerts and print the details of the last 10 alerts in a tabular format.
+
+
+.EXAMPLE
+./Get-NB-Jobs.ps1 -nbmaster "nb-master.example.com" -username "administrator" -password "secret"
```
Comment (kk-github-personal) [inline snippets/powershell/Get-NB-Alerts.ps1]: Usage points to GET-NB-jobs.

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (kk-github-personal) [inline .vscode/settings.json]: Does this file need to be checked-in?

File: snippets/powershell/Get-NB-Alerts.ps1 [hunk 3]
```diff
@@ -0,0 +1,131 @@
+﻿<#
+
+.SYNOPSIS
+This sample script demonstrates the use of NetBackup REST API for listing the alerts.
+
+.DESCRIPTION
+This script will get a list of netbackup alerts and print the details of the last 10 alerts in a tabular format.
```
Comment (kk-github-personal) [inline snippets/powershell/Get-NB-Alerts.ps1]: Do you mean 10 latest alerts?

Comment (kk-github-personal) [reply snippets/powershell/Get-NB-Alerts.ps1]: You might also want to talk about what type of alerts the API returns.

File: .vscode/settings.json (hunk 2 above)
Comment (ashutosh1211) [reply .vscode/settings.json]: I will remove this.

File: snippets/python/api_requests.py [hunk 4]
```diff
@@ -54,5 +54,22 @@ def get_netbackup_jobs(jwt, base_url):
...
+	#	"page[limit]": 100, 				#This changes the default page size to 100
+	#  	"filter":"subCategory eq 'VMWARE'"  # This adds a filter to only show the alerts for job failures for VMWARE
+	}
+	
+	print("performing GET  on {}\n".format(url))
+	
+	resp = requests.get(url, headers=headers, params=query_params, verify=False)
+	
+	if resp.status_code != 200:
+		raise Exception('Alert API failed with status code {} and {}'.format(resp.status_code, resp.json()))
```
Comment (ghost) [inline snippets/python/api_requests.py:73]: what will be the resp.json() in case of failure?

**Comment by ghost** [comment]
looks okay..

File: snippets/python/api_requests.py [hunk 5]
```diff
@@ -54,5 +54,22 @@ def get_netbackup_jobs(jwt, base_url):
...
+	url = base_url + "/manage/alerts"
+	headers = {'Content-Type': content_type, 'Authorization': jwt}
+	query_params = {
+	#	"page[limit]": 100, 				#This changes the default page size to 100
+	#  	"filter":"subCategory eq 'VMWARE'"  # This adds a filter to only show the alerts for job failures for VMWARE
+	}
+	
+	print("performing GET  on {}\n".format(url))
+	
+	resp = requests.get(url, headers=headers, params=query_params, verify=False)
```
Comment (ghost) [inline snippets/python/api_requests.py:70]: make sure empty query param works..

**Review by ghost — APPROVED** [review]
minor comments ..can be checked later.. Rest looks ok

**Pull request merged**